### PR TITLE
New languages features

### PR DESCRIPTION
**Pull request by tedoc2000**
New language support:
1. Tuple support
2. Ability to just parse an expression
3. import statement support

Switched over to snake_case for all parser rules to be consistent

### CODE REVIEW

File: grammar/SignalFlowV2Parser.g4 [hunk 1]
```diff
@@ -124,6 +128,14 @@ atom
   | FALSE
   ;
 
+paren_atom
+  : OPEN_PAREN testlist? CLOSE_PAREN
+  ;
+
+testlist:
+  test ( COMMA test)* COMMA?
```
Comment (mpetazzoni) [inline grammar/SignalFlowV2Parser.g4]: Is the space after the paren required?

File: grammar/SignalFlowV2Parser.g4 [hunk 2]
```diff
@@ -10,6 +10,10 @@ program
   : ( NEWLINE | statement )* EOF
   ;
 
+eval_input
+ : testlist NEWLINE* EOF
```
Comment (mpetazzoni) [inline grammar/SignalFlowV2Parser.g4:14]: Is that to parse just a boolean expression?

Comment (tedoc2000) [reply grammar/SignalFlowV2Parser.g4:14]: This is to parse any expression (think eval in python)

File: grammar/SignalFlowV2Parser.g4 [hunk 3]
```diff
@@ -124,6 +170,14 @@ atom
   | FALSE
   ;
 
+paren_atom
```
Comment (rajeshraman) [inline grammar/SignalFlowV2Parser.g4]: This should be called `tupleExpr` or some such.

**Review by rajeshraman — APPROVED** [review]

**Comment by tedoc2000** [comment]
switched to snake_case

**Review by mpetazzoni — APPROVED** [review]
👍

**Pull request merged**